server/config.yaml.sample

```diff
@@ -7,6 +7,17 @@ poolingConfig:
 googleAnalytics:
   enabled: true
 
+demo:
+  enabled: true
+  type: [pokeshop, otel]
```

Comment:
if you want to allow both demo systems to coexist simultaneously, I think a more clear approach would be:
```
demo:
  enable: [pokeshop, otel]
// ...
```

Reply:
Done